src/acl.c

```diff
@@ -560,6 +560,11 @@ void aclCommand(client *c) {
         }
     } else if (!strcasecmp(sub,"getuser") && c->argc == 3) {
         user *u = ACLGetUserByName(c->argv[2]->ptr,sdslen(c->argv[2]->ptr));
+        if (u == NULL) {
+            addReplyNull(c);
```

Comment:
I just wonder, what is better to return Null or "Not Exist" Message?
(just question.)